src/pdfkernel.cpp

```diff
@@ -267,11 +267,17 @@ wxPdfDocument::SelectFont(const wxPdfFont& font, int style, double size, bool se
     }
     if (wxPdfFontManager::GetFontManager()->InitializeFontData(font))
     {
-      wxString fontKey = wxString::Format(wxS("%s[%s]"), font.GetName().Lower().c_str(), font.GetEncoding().Lower().c_str());
+      wxString fontKey = font.GetName().Lower();
```

Comment:
I'd suggest adding a (inline) function for constructing font key from font instead of repeating the same lines twice.

Also, could calling `fontKey.reserve(font.GetName().length() + whatever-the-max-encoding-length-is)` be even better, to avoid reallocation?